src/Bicep.Cli/Arguments/FormatArguments.cs

```diff
@@ -141,14 +161,33 @@ public FormatArguments(string[] args, IOContext io) : base(Constants.Command.For
                         {
                             throw new CommandLineException($"The --insertFinalNewline parameter cannot be specified twice");
                         }
-                        InsertFinalNewline = true;
+
+                        if (bool.TryParse(args[i + 1], out var insertFinalNewline))
+                        {
+                            InsertFinalNewline = insertFinalNewline;
+                            i++;
+                        }
+                        else
+                        {
+                            InsertFinalNewline = insertFinalNewline;
```

Comment:
This clause is unclear.  insertFinalNewLine is false if TryParse fails?  If so, I'd much rather see "false" used than using what feels like an undefined variable.

Reply:
Although I'd have expected to see an assignment to true here.

Reply:
Oops, this is a copy-n-paste error...good catch. The value should be set to true. Fixed the code and added a comment to explain this logic.